### PR TITLE
Release 4.8.2

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

**Comment by robkooper** [comment]
can this be executed in a  container notebook instead of @Vismayak machine, notice the path to the files. I want to make sure this is what the user will see when they execute and it works as expected in the notebook docker image.

**Comment by navarroc** [comment]
> can this be executed in a container notebook instead of @Vismayak machine, notice the path to the files. I want to make sure this is what the user will see when they execute and it works as expected in the notebook docker image.

I'm not sure what you mean by executed in a container notebook. Do you mean executed on something like IN-CORE lab so the paths are similar to what a user would see?

**Comment by robkooper** [comment]
correct, and also that would make sure it is the same environment the user will use, not a conda environment that is on the developers machine.

**Comment by longshuicy** [comment]
> correct, and also that would make sure it is the same environment the user will use, not a conda environment that is on the developers machine.

sorry late to the game. What do we need to do for that?

**Comment by navarroc** [comment]
IMO - we open an issue and for the next release, we run all the example testbed notebooks on incore-lab to get a consistent user path.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to the INCORE documents generated by Sphinx package will be
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [4.8.2] - 2024-02-08
```
Comment (ywkim312) [inline CHANGELOG.md:8]: is it 02-08

**Review by ywkim312 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (longshuicy) [reply CHANGELOG.md:8]: you mean 7th? will do

Comment (ywkim312) [reply CHANGELOG.md:8]: yes

**Review by ywkim312 — APPROVED** [review]

**Comment by longshuicy** [comment]
> IMO - we open an issue and for the next release, we run all the example testbed notebooks on incore-lab to get a consistent user path.

Ah I think I understand. Some of the notebook output cell has local paths? 
Yes agree we should use incore-lab or a "anounmous" environment to run the notebooks that have outputs. 
Will create a task.

**Review by ylyangtw — APPROVED** [review]

**Pull request merged**